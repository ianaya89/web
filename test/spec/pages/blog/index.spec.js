import { shallowMount } from '@vue/test-utils'
import Index from '@/pages/index'

describe('Blog Index page', () => {
  it('renders desired sections', () => {
    const wrapper = shallowMount(Index)

    expect(wrapper.element).toMatchSnapshot()
  })
})
