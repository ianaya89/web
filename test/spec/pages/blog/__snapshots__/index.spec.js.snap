// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blog Index page renders desired sections 1`] = `
<div
  class="public-page overflow-hidden"
>
  <div
    class="min-h-screen w-full flex relative"
  >
    <div
      class="w-full h-screen flex absolute top-0"
    >
      <transition-stub
        appear="true"
        name="clip-path-slide-up"
      >
        <div
          class="w-2/3 xl:w-3/5 h-full hero-bg-img-01 bg-cover bg-right bg-no-repeat"
        />
      </transition-stub>
       
      <transition-stub
        appear="true"
        name="clip-path-slide-up-delayed"
      >
        <div
          class="w-1/3 xl:w-1/5 h-full hero-bg-img-02"
        />
      </transition-stub>
       
      <transition-stub
        appear="true"
        name="clip-path-slide-up-delayed-2"
      >
        <div
          class="hidden xl:block w-1/5 h-full hero-bg-img-03"
        />
      </transition-stub>
    </div>
     
    <div
      class="w-4 sm:w-16 xl:w-1/5 pl-4 xl:pl-8 z-10"
    >
      <div
        class="w-3/5 relative flex flex-col xl:border-r border-light-line pt-24 h-full"
      >
        <div
          class="scroll-info hidden xl:block"
        >
          <span>
            
            TRANSLATE:public.scroll
          
          </span>
        </div>
      </div>
    </div>
     
    <div
      class="xl:w-3/5 flex z-10"
    >
      <div
        class="border-l border-light-line sm:pt-24 text-white flex flex-wrap items-center"
      >
        <div
          class="pt-16"
        >
          <div
            class="pl-8"
          >
            <h1
              class="uppercase text-h2 sm:text-h1 max-w-md pr-12 leading-none"
            >
              
              TRANSLATE:landing.team.title
            
            </h1>
             
            <transition-stub
              appear="true"
              name="fade-up-delayed"
            >
              <h3
                class="mt-2 mb-16 text-h6 xl:text-h5 font-serif max-w-none sm:max-w-sm pr-8 xl:pr-20 leading-normal"
              >
                
                TRANSLATE:landing.team.subtitle
              
              </h3>
            </transition-stub>
          </div>
           
          <transition-stub
            appear="true"
            name="fade-up-delayed"
          >
            <animated-button-stub
              showarrow="true"
              title="TRANSLATE:landing.team.title_signup"
            />
          </transition-stub>
        </div>
      </div>
    </div>
     
    <div
      class="w-1/5 hidden xl:flex pr-12 z-10"
    >
      <div
        class="w-full border-r border-light-line"
      />
    </div>
  </div>
   
  <sponsor-team-promo-stub
    class="z-10 relative"
  />
   
  <!---->
   
  <div
    class="xl:min-h-75vh w-full flex bg-white mt-4 sm:mt-0"
  >
    <div
      class="w-1/5 hidden xl:flex"
    >
      <div
        class="w-4/5 ml-2"
      />
    </div>
     
    <div
      class="w-full xl:w-4/5 feature-section lg:border-l border-grey-l3"
    >
      <div
        class="w-full block sm:flex"
      >
        <div
          class="sm:w-1/3 px-8 sm:pl-20 xl:pl-10 pr-6"
        >
          <h2
            class="section-title mt-24 sm:mt-56 xl:mt-32 max-w-xs"
          >
            
            TRANSLATE:landing.team.how_it_works.title
          
          </h2>
        </div>
         
        <div
          class="sm:w-2/3 sm:mr-8 xl:mr-64 sm:border-l lg:border-r border-grey-l3"
        >
          <appear-transition-stub>
            <div
              class="how-to-section sm:mt-48 xl:mt-32"
            >
              <h3
                class="uppercase text-h4 border-b-8 border-grey-l3 pb-1 inline"
              >
                
                TRANSLATE:landing.team.how_it_works.sign_up.title
              
              </h3>
               
              <p
                class="how-to-text"
              >
                
                TRANSLATE:landing.team.how_it_works.sign_up.body
              
              </p>
            </div>
          </appear-transition-stub>
           
          <appear-transition-stub>
            <div
              class="how-to-section"
            >
              <h3
                class="uppercase text-h4 border-b-8 border-grey-l3 pb-1 inline"
              >
                
                TRANSLATE:landing.team.how_it_works.upload.title
              
              </h3>
               
              <p
                class="how-to-text"
              >
                
                TRANSLATE:landing.team.how_it_works.upload.body
              
              </p>
            </div>
          </appear-transition-stub>
           
          <appear-transition-stub>
            <div
              class="how-to-section"
            >
              <h3
                class="uppercase text-h4 border-b-8 border-grey-l3 pb-1 inline"
              >
                
                TRANSLATE:landing.team.how_it_works.connections.title
              
              </h3>
               
              <p
                class="how-to-text"
              >
                
                TRANSLATE:landing.team.how_it_works.connections.body
              
              </p>
            </div>
          </appear-transition-stub>
           
          <appear-transition-stub>
            <div
              class="how-to-section"
            >
              <h3
                class="uppercase text-h4 border-b-8 border-grey-l3 pb-1 inline"
              >
                
                TRANSLATE:landing.team.how_it_works.deals.title
              
              </h3>
               
              <p
                class="how-to-text"
              >
                
                TRANSLATE:landing.team.how_it_works.deals.body
              
              </p>
            </div>
          </appear-transition-stub>
           
          <div
            class="px-8 sm:px-0"
          >
            <app-button-stub
              btnclass="primary-lg"
              class="w-full sm:w-auto"
              showarrow="true"
              title="TRANSLATE:general.get_started"
            />
          </div>
           
          <iframe
            allowfullscreen="allowfullscreen"
            class="w-full mt-16 mb-24 lg:mb-48 h-64 xl:h-128"
            frameborder="0"
            src="https://www.youtube-nocookie.com/embed/iR9WK373sQo?controls=0&rel=0"
            title="SPONSOR.ONLINE video"
          />
        </div>
      </div>
    </div>
  </div>
   
  <div
    class="w-full flex bg-white mt-4 sm:mt-16 xl:mt-0 mb-80 sm:mb-0"
  >
    <div
      class="w-1/5 hidden xl:flex"
    >
      <div
        class="w-4/5 ml-2"
      />
    </div>
     
    <div
      class="w-full xl:w-4/5 feature-section lg:border-l border-grey-l3"
    >
      <animated-features-stub
        body="TRANSLATE:landing.team.under_hood.body"
        forceanimate="true"
        items="[object Object],[object Object],[object Object]"
        title="TRANSLATE:landing.team.under_hood.title"
      />
    </div>
  </div>
   
  <div
    class="w-full flex flex-wrap"
  >
    <div
      class="w-full xl:w-2/5 lg:border-r border-grey-l3 flex pt-24 sm:pt-40 xl:pt-48"
    >
      <div
        class="w-full xl:w-auto xl:max-w-xxs xl:mx-auto px-8 xl:px-0"
      >
        <div
          class="flex flex-wrap sm:flex-no-wrap xl:flex-wrap"
        >
          <div
            class="pr-8"
          >
            <tween-content-stub
              duration="1200"
              localqueryselector="h1"
              startvalue="0"
            >
              <h1
                class="text-5xl lg:text-6xl font-medium mt-8 xl:mt-0 leading-none"
              >
                
                TRANSLATE:landing.team.save_money.annual_sponsorship.title
              
              </h1>
            </tween-content-stub>
             
            <p
              class="tween-label"
            >
              
              TRANSLATE:landing.team.save_money.annual_sponsorship.body
            
            </p>
          </div>
           
          <div
            class="pr-8"
          >
            <tween-content-stub
              duration="1000"
              localqueryselector="h1"
              startvalue="0"
            >
              <h1
                class="text-5xl lg:text-6xl font-medium mt-8 leading-none"
              >
                
                TRANSLATE:landing.team.save_money.revenue.title
              
              </h1>
            </tween-content-stub>
             
            <p
              class="tween-label"
            >
              
              TRANSLATE:landing.team.save_money.revenue.body
            
            </p>
          </div>
           
          <div
            class="pr-8"
          >
            <tween-content-stub
              duration="1000"
              localqueryselector="h1"
              startvalue="0"
            >
              <h1
                class="text-5xl lg:text-6xl font-sans font-medium mt-8 leading-none"
              >
                
                TRANSLATE:landing.team.save_money.agency_fees.title
              
              </h1>
            </tween-content-stub>
             
            <p
              class="tween-label"
            >
              
              TRANSLATE:landing.team.save_money.agency_fees.body
            
            </p>
          </div>
        </div>
         
        <app-button-stub
          btnclass="primary-lg"
          class="mt-10 w-full xl:w-auto hidden"
          title="Add your team"
        />
         
        <ul
          class=" pt-24 xl:pt-32 pb-10"
        >
          <li
            class="inline uppercase font-medium color-grey-l1 text-h4 mr-4 pr-2 text-black border-b-8 border-grey-l3"
          >
            
            TRANSLATE:landing.team.save_money.costs.label
          
          </li>
        </ul>
         
        <ul
          class="saving-list"
        >
          <li
            class="sponsor-item"
          >
            
            TRANSLATE:landing.team.save_money.costs.agencies.sponsor
          
          </li>
           
          <li
            class="agencies-item"
          >
            
            TRANSLATE:landing.team.save_money.costs.agencies.agencies
          
          </li>
        </ul>
      </div>
    </div>
     
    <appear-transition-stub
      percentageoffset="0.4"
    >
      <div
        class="xl:w-4/5 pt-24 xl:pt-48 relative flex flex-col-reverse xl:flex-col xl:pb-16"
      >
        <div
          class="px-8"
        >
          <div
            class="max-w-xs -mt-2 pt-24 sm:pt-48 xl:pt-4"
          >
            <h1
              class="uppercase text-6xl font-medium leading-none"
            >
              
              TRANSLATE:landing.team.save_money.title
            
            </h1>
          </div>
           
          <p
            class="max-w-md font-medium text-h7 text-grey my-6"
          >
            
            TRANSLATE:landing.team.save_money.p1
          
          </p>
           
          <h1
            class="text-h3 xl:text-h2 font-medium leading-tight"
          >
            
            TRANSLATE:landing.team.save_money.subtitle1
          
          </h1>
           
          <p
            class="font-medium text-h7 max-w-md text-grey mt-2 mb-8"
          >
            
            TRANSLATE:landing.team.save_money.p2
          
          </p>
           
          <div
            class="max-w-sm"
          >
            <h1
              class="text-h3 xl:text-h2 font-medium"
            >
              
              TRANSLATE:landing.team.save_money.subtitle2
            
            </h1>
          </div>
        </div>
         
        <img
          alt="TRANSLATE:landing.team.save_money.chart_alt"
          class="save-chart-image -mt-8 sm:-mt-24 xl:-mt-8 xl:-ml-48 pb-16 absolute xl:relative"
          src="\\"save__chart.png\\""
        />
      </div>
    </appear-transition-stub>
  </div>
   
  <div
    class="w-full flex flex-wrap xl:-mt-2"
  >
    <div
      class="w-4/5 xl:w-2/5 border-r border-grey-l3 flex px-8 xl:px-0 pb-8 xl:pb-0"
    >
      <div
        class="w-full xl:w-auto xl:max-w-xxs xl:mx-auto pt-24 xl:pt-32"
      >
        <div
          class="flex flex-wrap sm:flex-no-wrap xl:flex-wrap justify-between"
        >
          <div
            class="pr-8 xl:w-full"
          >
            <tween-content-stub
              duration="1000"
              localqueryselector="h1"
              startvalue="0"
            >
              <h1
                class="text-5xl lg:text-6xl font-medium leading-tight mt-8 xl:mt-0"
              >
                
                TRANSLATE:landing.team.deals.clubs.title
              
              </h1>
            </tween-content-stub>
             
            <p
              class="tween-label"
            >
              
              TRANSLATE:landing.team.deals.clubs.body
            
            </p>
          </div>
           
          <div
            class="pr-8 xl:w-full"
          >
            <tween-content-stub
              duration="1200"
              localqueryselector="h1"
              startvalue="0"
            >
              <h1
                class="text-5xl lg:text-6xl font-sans font-medium mt-8"
              >
                
                TRANSLATE:landing.team.deals.brands.title
              
              </h1>
            </tween-content-stub>
             
            <p
              class="tween-label"
            >
              
              TRANSLATE:landing.team.deals.brands.body
            
            </p>
          </div>
           
          <div
            class="pr-8 xl:w-full"
          >
            <tween-content-stub
              duration="1000"
              localqueryselector="h1"
              startvalue="0"
            >
              <h1
                class="text-5xl lg:text-6xl font-medium mt-8"
              >
                
                TRANSLATE:landing.team.deals.reg_brands.title
              
              </h1>
            </tween-content-stub>
             
            <p
              class="tween-label"
            >
              
              TRANSLATE:landing.team.deals.reg_brands.body
            
            </p>
          </div>
        </div>
      </div>
    </div>
     
    <appear-transition-stub
      percentageoffset="0.5"
    >
      <div
        class="w-full relative pt-24 xl:pt-32 pb-32 xl:pb-32"
      >
        <div
          class="max-w-md -mt-2 px-8 pt-4"
        >
          <h1
            class="uppercase text-h2 sm:text-6xl font-medium leading-none"
          >
            
            TRANSLATE:landing.team.deals.title
          
          </h1>
        </div>
         
        <p
          class="max-w-md font-medium text-h6 xl:text-h7 text-grey mt-6 mb-16 px-8"
        >
          
          TRANSLATE:landing.team.deals.body
        
        </p>
         
        <div
          class="px-8 xl:px-0 "
        >
          <app-button-stub
            btnclass="primary-lg"
            class="w-full sm:w-auto"
            showarrow="true"
            title="TRANSLATE:general.lets_get_started"
          />
        </div>
      </div>
    </appear-transition-stub>
  </div>
   
  <div
    class="w-full flex flex-wrap xl:-mt-2"
  >
    <div
      class="w-full lg:w-1/2 xl:w-2/5 border-r border-grey-l3 flex pb-8 xl:pb-0 xl:justify-end"
    >
      <appear-transition-stub
        class="relative w-full xl:w-auto"
        percentageoffset="0.4"
      >
        <iframe
          allowfullscreen="allowfullscreen"
          class="w-full xl:w-128 h-80 sm:h-128 lg:h-80"
          frameborder="0"
          src="https://www.youtube.com/embed/2jcX3JJ9udA?controls=0&rel=0"
          title="SPONSOR.ONLINE video"
        />
      </appear-transition-stub>
    </div>
     
    <appear-transition-stub
      percentageoffset="0.5"
    >
      <div
        class="w-full lg:w-1/2 lg:w-full relative sm:pt-8 lg:pt-16 xl:pt-32 pb-32 xl:pb-32"
      >
        <div
          class="pl-8 lg:pt-16 pr-2 sm:flex items-end"
        >
          <div
            class="xl:mb-16"
          >
            <p
              class="font-sans font-medium text-grey-d1 text-h7"
            >
              
              Kerstin Lutz, SPONSOR.ONLINE Advisory Board
            
            </p>
             
            <p
              class="font-sans text-grey font-normal text-h7"
            >
              
              Managing Director Partnership Management, TEAM Marketing
            
            </p>
             
            <app-button-stub
              btnclass="link"
              class="mt-8 w-auto"
              showarrow="true"
              title="Meet our team"
            />
          </div>
        </div>
      </div>
    </appear-transition-stub>
  </div>
</div>
`;
